### ISSUE TITLE
Errors using Bakta annotated assemblies as input to panaroo

### ISSUE THREAD

**Issue by matbeale** [gtonkinhill/panaroo#177]
Hi Gerry and co,

I've been looking at using Bakta annotated assemblies as an alternative input for panaroo, but am encountering a few issues. Firstly, the standard bakta `.gff3` don't have the concatenated `.fna` on the end (easy to fix). More of an issue is errors around `invalid gene! file - id:` which are causing panaroo to exit. I identified some slightly odd locus_tag headers (containing '%' characters), but the remaining errors in other lines are not obvious...

Happy to provide example files (or point to farm paths). 

Many thanks,
Mat

**Comment by iqbal-lab** [comment]
Cc @Danderson123

**Comment by gtonkinhill** [comment]
Hi Mat,

I would be keen to get Panaroo working with Bakta output. If you have a small set of example files that would be great! It would be good if you could highlight a few of the annotations that fail without an obvious reason.

A temporary fix while I look into it would be to include the `--remove-invalid-gene` flag which would skip over the annotations that failed the check.

**Comment by matbeale** [comment]
Hi Gerry,

Ok, so my log files are on farm at: 
```/lustre/scratch118/infgen/team216/mb29/other/Salmonella_Bongori__2022/panaroo/panaroo_Q20+bakta_20220709.log.e``` and ```panaroo_Q20+bakta_20220709.log.e```. 

and an example of a failing gff error would be: 
```invalid gene! file - id:  /lustre/scratch118/infgen/team216/mb29/other/Salmonella_Bongori__2022/panaroo/assemblies-bakta/GCA_000025745.1_ASM2574v1_genomic.gff  -  AONEMB_20815```

There is a dataset of ~127 bakta annotated public genomes at: 
``` /lustre/scratch118/infgen/team216/mb29/other/Salmonella_Bongori__2022/panaroo/assemblies-bakta/``` 
which I've been working with. 

Once I realised I had this issue, I reannotated everything with prokka, leading to a successful panaroo run on the exact same settings, but I'll take a look at `--remove-invalid-gene` and hopefully that will provide an alternate fix for bakta (it's not a lot of genes coming up). 

Many thanks,
Mat

**Comment by gtonkinhill** [comment]
Hi Mat,

Thanks very much for linking the files. I have updated Panaroo to include the option of providing separate GFF and FASTA files. I also added support for genbank files. When I ran the latest version using the `--remove-invalid-gene` on the set of files you provided it seemed to run okay.

Would it be possibly for you to double check the update works for you? The dev version can be installed by running 

```
pip install git+https://github.com/gtonkinhill/panaroo@devel
```
